src/main/java/lesson17/Main.java

```diff
@@ -7,7 +7,18 @@
     public static void main(String[] args) throws SQLException {
```

Comment:
Why you wrout method main? Where are tests ????????

Reply:
Sorry. Alredy add it in 2ond commit